grammar.y

```diff
@@ -1357,6 +1449,9 @@ paren_update_list:
 update_expression:
   column_name '=' expr
   {
+    if isRowID($1.Name) {
```

Comment:
Block references to `ROWID` on an `UPDATE` (this is the classical `UPDATE` syntax)